app/models/notification_setting.rb

```diff
@@ -28,6 +28,18 @@ class NotificationSetting < ApplicationRecord
   enum setting_type: NOTIFICATION_TYPES
   belongs_to :user
 
+  def self.get_stream_kitsu_notification_type(type, stream_mentions, feed_id)
```

Comment:
Generally method names prefixed with `get_` are frowned upon in Ruby, as we don't draw a distinction between property getters and method calls.  Also this method name is quite a mouthful lol

Though truthfully I have no ideas on a better name... `type_for_activity`? 🤔 Nothing really quite seems to describe it terribly well